packages/core/src/components/Banner.tsx

```diff
@@ -0,0 +1,204 @@
+import * as React from "react";
+import {
+  Button,
+  Text,
+  View,
+  ViewStyle,
+  StyleSheet,
+  StyleProp,
+  Animated,
+} from "react-native";
+import Surface from "./Surface";
+import type { IconSlot } from "../interfaces/Icon";
+import shadow from "../styles/shadow";
+import { withTheme } from "../theming";
+import type { Theme } from "../styles/DefaultTheme";
+
+const ELEVATION = 1;
+const DEFAULT_MAX_WIDTH = 960;
+
+type Props = {
+  initiallyVisible: boolean;
```

Comment:
For non-optional props, can you set a default when you destructure below?

Reply:
Definitely